.github/workflows/test_on_push.yml

```diff
@@ -272,8 +260,7 @@ jobs:
       - name: Run integration tests for ${{ matrix.os }} with Python ${{ matrix.python-version }}
         run: python -m nox -s integration
 
-# Runs only on Ubuntu with Python 3.12. Skips IDAKLU module compilation
-# for speedups, which is already tested in other jobs.
+  # Skips IDAKLU module compilation for speedups, which is already tested in other jobs.
```

Comment:
We should be able to bump this job back to Python 3.12 now

Reply:
Done